keeperapi/src/vault.ts

```diff
@@ -73,6 +73,7 @@ export type DRecord = {
     shared: boolean
     clientModifiedTime: number
     extra?: any
+    udata?: any
```

Comment:
At this point, I have limited knowledge in the `udata`'s data type. What I know for sure now is that it's an object and has `file_ids: string[]` field. I can use the explicit data type here. But the downside is that if we find out more fields should be supported later, we need to make another PR to address that.

So I think using the `any` type is okay? Probably the same reasons were there for the `extra` property as well. Let me know if you see some concerns on it.

I'm not sure if this is a good practice to use `any` type here. Now, I have limited knowledge on its actual data type. But what I know for sure for sure is that `udata` is an object having `file_ids: string[]`.